pyproject.toml

```diff
@@ -33,6 +33,7 @@ django-stubs = "^1.12.0"
 django-debug-toolbar = "^3.7.0"
 django-linear-migrations = "^2.5.1"
 pre-commit = "^2.20.0"
+django-browser-reload = "^1.6.0"
```

Comment:
Let's do only one thing with this PR. The PR should focus only on the model factories. We can add `django-browser-reload` in a separate, smaller pull request.

Reply:
okay

Reply:
would make a new push request

Reply:
have made a new PR, and removed the additions of changes in issue #1346 making it focused on the model factories only